File: src/Backend/src/models/Comentario.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/config");

const Comentario = sequelize.define('comentario', {
    nome: {
        type: DataTypes.STRING,
        allowNull: false
    },
    descricao: {
        type: DataTypes.STRING,
        allowNull: false
    },
    nota: {
        type: DataTypes.INTEGER,
        allowNull: false
    }
});

module.exports = Comentario;

File: src/Frontend/app/src/main/java/br/com/aula/text/Comentario.java
package br.com.aula.text;

public class Comentario {
    private String nome;
    private String descricao;

    public Comentario(String nome, String descricao) {
        this.nome = nome;
        this.descricao = descricao;
    }

    public String getNome() {
        return nome;
    }

    public String getDescricao() {
        return descricao;
    }
}